aws_lambda_builders/workflows/nodejs_npm_esbuild/esbuild.py

```diff
@@ -296,3 +297,14 @@ def _get_explicit_file_type(self, entry_point, entry_path):
                 return entry_point + ext
 
         raise ActionFailedError("entry point {} does not exist".format(entry_path))
+
+    @staticmethod
+    def _get_dash_delimited_argument(arg: str):
```

Comment:
nit: This could have a `str` return type hint

Reply:
Nice catch!